.solhint.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "solhint:recommended",
+  "plugins": ["prettier"],
+  "rules": {
+    "code-complexity": ["error", 8],
+    "compiler-version": ["error", ">=0.8.4"],
+    "func-visibility": ["error", {"ignoreConstructors": true}],
+    "max-line-length": ["off", 120],
```

Comment:
Tried it and reverted it again because the 120 characters are often reached already for solidity `import` statements. I am fine with loosening the style regarding comment length.